### PR TITLE
Initial

### PR DESCRIPTION
**Pull request by notjames**
all the things

### CODE REVIEW

File: op.yml [hunk 1]
```diff
@@ -1,3 +1,41 @@
+---
 name: github.com/opspec-pkgs/docker.push.localimage
-description: push docker image that is stored on host machine
+description:
+  Builds image that stores on host machines docker image store.
```
Comment (chrisdostert) [inline op.yml]: was this meant to be push?

Comment (notjames) [reply op.yml]: Yikes! Yes. I'll fix and re-submit. Thanks for the catch!

**Comment by chrisdostert** [comment]
should I merge this? If so, should I also release it?

**Comment by notjames** [comment]
Yes please...and yes. You rock!

**Comment by chrisdostert** [comment]
Thx, so do u! One last thing u might choose to do before we snap a release:
u might want to snag the docker icon from https://github.com/opspec-pkgs/docker.image.build/blob/master/icon.svg (or other docker op) and re-build so it gets included in the readme

**Comment by notjames** [comment]
I think I did that right. Please let me know.

**Comment by chrisdostert** [comment]
it looks like you did it right but the .svg just isn't valid.  The .svg content is just xml text so you can either open raw view on github & copy/paste or download the file; either should do

**Comment by notjames** [comment]
hmm. I downloaded the file via `curl` but I'll do it again and re-push.

**Comment by chrisdostert** [comment]
check difference
good: https://raw.githubusercontent.com/opspec-pkgs/docker.image.build/master/icon.svg
bad: https://raw.githubusercontent.com/opspec-pkgs/docker.push.localimage/9b55466e57d277d96767aade3b88ed8210738c9e/icon.svg

**Comment by notjames** [comment]
try that? If that doesn't work, I have one more thing I can try.

**Comment by chrisdostert** [comment]
arg; still not valid. Check https://github.com/opspec-pkgs/docker.push.localimage/blob/7f6130f69c2f606596c303d1e36ead5bd1a074ef/icon.svg to test. You should just be able to copy text out of the other .svg and be golden

**Pull request merged**